master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -184,6 +185,9 @@ spec:
             # Disable IPv6 on Kubernetes.
             - name: FELIX_IPV6SUPPORT
               value: "false"
+            # Set MTU for tunnel device
+            - name: FELIX_IPINIPMTU
+              value: 1440
```

Comment:
I thought the default value set by Felix was 1400 - is there a reason to change it to 1440?